tests/pkg_micro-ecc/main.c

```diff
@@ -0,0 +1,135 @@
+/*-
+ * Copyright 2014 Kenneth MacKay
+ * Copyright 2014 Frank Holtz
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS'' AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+ * OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+ * SUCH DAMAGE.
+ *
+ */
+
+/**
+ * @file
+ * @brief       Check if the micro-ecc builds and working
+ *
+ * @author      Frank Holtz <frank-riot2015@holtznet.de>
+ *
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <uECC.h>
+
+#define TESTROUNDS	16
+
+void vli_print(uint8_t *p_vli, unsigned int p_size)
+{
+    while (p_size) {
+        printf("%02X ", (unsigned)p_vli[p_size - 1]);
+        --p_size;
+    }
+}
+
+int main(void)
+{
+    printf("micro-ecc compiled!\n");
+
+    int i, errorc = 0;
+
+    uint8_t l_private1[uECC_BYTES];
+    uint8_t l_private2[uECC_BYTES];
+
+    uint8_t l_public1[uECC_BYTES * 2];
+    uint8_t l_public2[uECC_BYTES * 2];
+
+    uint8_t l_secret1[uECC_BYTES];
+    uint8_t l_secret2[uECC_BYTES];
+
+    uint8_t l_hash[uECC_BYTES];
+
+    uint8_t l_sig[uECC_BYTES * 2];
+
+
+    printf("Testing %d random private key pairs and signature\n", TESTROUNDS);
+
+    for (i = 0; i < TESTROUNDS; ++i) {
+        printf(".");
+#ifdef CPU_NATIVE
+        fflush(stdout);
+#endif
```

Comment:
At least on my system this is superfluous, the dots also get printed continuously without this line.
Even if: I wouldn't make an exception for native here.